src/composables/usePool.ts

```diff
@@ -33,13 +29,23 @@ import { AnyPool, Pool, PoolToken, SubPool } from '@/services/pool/types';
 import { hasBalEmissions } from '@/services/staking/utils';
 import { uniq, uniqWith, cloneDeep } from 'lodash';
 
+const POOLS = configService.network.pools;
+
 /**
  * METHODS
  */
 export function addressFor(poolId: string): string {
   return getAddress(poolId.slice(0, 42));
 }
 
+export function metadata(poolId: string): PoolMetadata | undefined {
```

Comment:
Actually, there is already a function in this file called `poolMetadata` which is a bit more robust but does the same thing. We sometimes need to fetch metadata for pools that are not on the current network, e.g. in the verbal list.

Either rename `poolMetadata` to `metadata` and replace usage or keep `poolMetadata` and replaces your usage of `metadata`.